pagopa/io-app
# feat: [IOBP-870] Add bonus & payments profile and super properties on mixpanel

## Short description
This PR adds tracking for super properties and profile properties in the bonus and payments domains. This tracks the number of saved payment methods, whether the user has an active CGN card, and the list of enabled welfare (idpay) names. The tracking is triggered upon opening the user’s wallet section.

## List of changes proposed in this pull request
- Added the property `CGN_STATUS`, `WELFARE_STATUS` and `SAVED_PAYMENT_METHOD`;
- Added a trigger to update the profile and super properties on mixpanel inside the ts/features/newWallet/saga/index.ts saga that fires the update when the all three requests are done (both in failure or in success);

## How to test
Open the wallet screen and check that the profile and super properties that are sent to mixpanel are populated as expected
